2021-03-14 20:13:17,244 INFO    MainThread:43562 [wandb_setup.py:_flush():69] setting env: {}
2021-03-14 20:13:17,244 INFO    MainThread:43562 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-14 20:13:17,244 INFO    MainThread:43562 [wandb_init.py:_log_setup():334] Logging user logs to /cbr/saish/PhD/DL/Assignment_1/wandb/run-20210314_201317-28rbif9l/logs/debug.log
2021-03-14 20:13:17,245 INFO    MainThread:43562 [wandb_init.py:_log_setup():335] Logging internal logs to /cbr/saish/PhD/DL/Assignment_1/wandb/run-20210314_201317-28rbif9l/logs/debug-internal.log
2021-03-14 20:13:17,245 INFO    MainThread:43562 [wandb_init.py:init():367] calling init triggers
2021-03-14 20:13:17,245 INFO    MainThread:43562 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'batch_size': 64, 'l_rate': 1e-05, 'optimizer': 'mgd', 'epochs': 5, 'activation': 'sigmoid', 'initializer': 'xavier', 'loss': 'cross_entropy'}
2021-03-14 20:13:17,245 INFO    MainThread:43562 [wandb_init.py:init():416] starting backend
2021-03-14 20:13:17,245 INFO    MainThread:43562 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-03-14 20:13:17,255 INFO    MainThread:43562 [backend.py:ensure_launched():123] starting backend process...
2021-03-14 20:13:17,259 INFO    MainThread:43562 [backend.py:ensure_launched():128] started backend process with pid: 43642
2021-03-14 20:13:17,260 INFO    MainThread:43562 [wandb_init.py:init():421] backend started and connected
2021-03-14 20:13:17,261 INFO    MainThread:43562 [wandb_init.py:init():461] updated telemetry
2021-03-14 20:13:17,261 INFO    MainThread:43562 [wandb_init.py:init():480] communicating current version
2021-03-14 20:13:18,181 INFO    MainThread:43562 [wandb_init.py:init():485] got version response 
2021-03-14 20:13:18,181 INFO    MainThread:43562 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-03-14 20:13:18,520 INFO    MainThread:43562 [wandb_init.py:init():518] starting run threads in backend
2021-03-14 20:13:20,540 INFO    MainThread:43562 [wandb_run.py:_console_start():1480] atexit reg
2021-03-14 20:13:20,541 INFO    MainThread:43562 [wandb_run.py:_redirect():1343] redirect: SettingsConsole.REDIRECT
2021-03-14 20:13:20,542 INFO    MainThread:43562 [wandb_run.py:_redirect():1348] Redirecting console.
2021-03-14 20:13:20,542 INFO    MainThread:43562 [redirect.py:install():213] install start
2021-03-14 20:13:20,542 INFO    MainThread:43562 [redirect.py:install():228] install stop
2021-03-14 20:13:20,543 INFO    MainThread:43562 [redirect.py:install():213] install start
2021-03-14 20:13:20,543 INFO    MainThread:43562 [redirect.py:install():228] install stop
2021-03-14 20:13:20,543 INFO    MainThread:43562 [wandb_run.py:_redirect():1394] Redirects installed.
2021-03-14 20:13:20,543 INFO    MainThread:43562 [wandb_init.py:init():542] run started, returning control to user process
2021-03-14 20:14:15,209 INFO    MainThread:43562 [wandb_run.py:_atexit_cleanup():1450] got exitcode: 255
2021-03-14 20:14:15,211 INFO    MainThread:43562 [wandb_run.py:_restore():1422] restore
2021-03-14 20:14:15,211 INFO    MainThread:43562 [redirect.py:uninstall():232] uninstall start
2021-03-14 20:14:15,212 INFO    MainThread:43562 [redirect.py:_stop():287] _stop: stdout
2021-03-14 20:14:15,215 INFO    stdout    :43562 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-03-14 20:14:15,215 INFO    stdout    :43562 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-03-14 20:14:15,215 INFO    MainThread:43562 [redirect.py:_stop():293] _stop closed: stdout
2021-03-14 20:14:15,215 INFO    MainThread:43562 [redirect.py:_stop():299] _stop joined: stdout
2021-03-14 20:14:15,215 INFO    MainThread:43562 [redirect.py:_stop():301] _stop rd closed: stdout
2021-03-14 20:14:15,215 INFO    MainThread:43562 [redirect.py:uninstall():236] uninstall done
2021-03-14 20:14:15,216 INFO    MainThread:43562 [redirect.py:uninstall():232] uninstall start
2021-03-14 20:14:15,216 INFO    MainThread:43562 [redirect.py:_stop():287] _stop: stderr
2021-03-14 20:14:15,216 INFO    stderr    :43562 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-03-14 20:14:15,216 INFO    stderr    :43562 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-03-14 20:14:15,216 INFO    MainThread:43562 [redirect.py:_stop():293] _stop closed: stderr
2021-03-14 20:14:15,216 INFO    MainThread:43562 [redirect.py:_stop():299] _stop joined: stderr
2021-03-14 20:14:15,216 INFO    MainThread:43562 [redirect.py:_stop():301] _stop rd closed: stderr
2021-03-14 20:14:15,216 INFO    MainThread:43562 [redirect.py:uninstall():236] uninstall done
2021-03-14 20:14:15,509 INFO    MainThread:43562 [wandb_run.py:_wait_for_finish():1572] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 33588
  total_bytes: 33588
}

2021-03-14 20:14:17,513 INFO    MainThread:43562 [wandb_run.py:_wait_for_finish():1572] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 76479
  total_bytes: 76479
}

